AppSearchSample/app/build.gradle

```diff
@@ -16,22 +16,23 @@
 
 plugins {
     id 'com.android.application'
-    id 'kotlin-android'
-    id 'kotlin-kapt'
+    id 'org.jetbrains.kotlin.android'
 }
 
+apply plugin: 'kotlin-kapt'
```

Comment:
How come we are using two different ways to apply plugins in this project (the plugins block and the apply plugin directive)?

Reply:
> How come we are using two different ways to apply plugins in this project (the plugins block and the apply plugin directive)?